src/index.js

```diff
@@ -1,11 +1,51 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import books from './reducers/books';
+import App from './components/App';
 import './index.css';
-import App from './App';
+
+const getRandomInt = () => Math.floor(Math.random() * Math.floor(214748364));
+
+const initialBookState = [
+  {
+    id: getRandomInt(),
+    title: 'Fahrenheit 451',
+    category: 'Sci-Fi',
+  },
+  {
+    id: getRandomInt(),
+    title: '1984',
+    category: 'Sci-Fi',
+  },
+  {
+    id: getRandomInt(),
+    title: 'Brave New World',
+    category: 'Sci-Fi',
+  },
+  {
+    id: getRandomInt(),
+    title: 'Dr. Frankesntein',
+    category: 'Sci-Fi',
+  },
+  {
+    id: getRandomInt(),
+    title: 'Dracula',
+    category: 'Horror',
+  },
+  {
+    id: getRandomInt(),
+    title: 'Metodos Numericos Para Ingenieria',
+    category: 'Learning',
+  },
+];
+
+const store = createStore(books, initialBookState);
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <Provider store={store}>
+    <App books={store.getState()} />
+  </Provider>,
```

Comment:
You should use `mapStateToProps` to get things from the store, you don't have to pass it directly. Also, you don't need to remove the `React.StrictMode`, you just need to wrap the `App` component with the `Provider` as shown below.
```suggestion
  <React.StrictMode>
    <Provider store={store}>
      <App />
    </Provider>
  </React.StrictMode>,
```